cdap-docs/developers-manual/source/data-exploration/tables.rst

```diff
@@ -0,0 +1,107 @@
+.. meta::
+    :author: Cask Data, Inc.
+    :copyright: Copyright © 2015 Cask Data, Inc.
+
+.. _table-exploration:
+
+============================================
+Table Exploration
+============================================
+
+A ``Table`` is a core Dataset. Unlike relational database tables where every
+row has the same schema, every row of a Table can have a different set of columns.
+Though Tables do not require a schema, in practice they are often written with an
+implicit schema. Column names are often strings, with a single data type used
+for all values in the same column. If you are using a Table in this way,
+you can set the schema Table property to enable exploration. The schema will be
+applied at read time, allowing you to run ad-hoc queries against the Table. 
+
+Requirements
+------------
+In order to explore a Table, your Table must meet a few requirements.
+
+* Columns names must be strings.
+
+* All column values for a specific column must be of the same type. For example, a value cannot be a string
+  in one row and an integer in another.
+
+* Column values must be of a primitive type.
+  A primitive type is one of boolean, int, long, float, double, bytes, or string. 
+
+* Column names must be valid Hive column names. This means they cannot be reserved keywords like 'drop'.
+  Please refer to the `Hive language manual <https://cwiki.apache.org/confluence/display/Hive/LanguageManual+DDL>`__
+  for more information about Hive.
+
+Creating an Explorable Table
+----------------------------
+
+When creating a ``Table`` in your application, if you set the schema table property, your Table
```

Comment:
Should that be "table's schema property", as the schema is a property of the table, and not the other way around?